viewer/src/main/webapp/viewer-html/components/FeatureInfo.js

```diff
@@ -66,7 +67,8 @@ Ext.define ("viewer.components.FeatureInfo",{
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_GET_FEATURE_INFO,this.onFeatureInfoStart,this);
         //listen to a extent change
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_CHANGE_EXTENT, this.onChangeExtent,this);
-        return this;        
+        this.config.viewerController.addListener(viewer.viewercontroller.controller.Event.ON_EDIT_SUCCESS, this.featureEdited, this);
```

Comment:
odd to use `this.config.viewerController` here while bove we use `this.getViewerController()`